README.md

```diff
@@ -56,7 +56,7 @@ Most useful available parameters:
 
 ## Reference
 
-###Classes 
+###Classes
```

Comment:
please add a space between `#*` and the headline
this makes the markdown more… platform… independent…